MapboxNavigation/NavigationViewController.swift

```diff
@@ -420,6 +433,7 @@ open class NavigationViewController: UIViewController, RouteMapViewControllerDel
         UIApplication.shared.applicationIconBadgeNumber = 0
     }
     
+    //MARK: - RouteMapViewControllerDelegate Methods
```

Comment:
A common practice in this codebase is to break out a separate extension and have it conform to the protocol. That way, the compiler will complain if any of the protocol’s required methods lies outside the extension.